.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,170 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - dev
+      - rb/parallelize-ci-workflow
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build_and_publish_image:
+    name: Build and Publish Image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: pgc-images.sbgenomics.com/d3b-bixu/open-pedcan
+          tags: |
+            type=raw,value=analysisjob
+            # Only tag the image with latest if we're building on the default
+            # branch (e.g., dev).
+            type=raw,value=latest,enable={{is_default_branch}}
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          registry: pgc-images.sbgenomics.com
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+  run_analysis:
+    name: Run Analysis
+    needs: build_and_publish_image
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
```

Comment:
[GitHub Actions build matrices](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) allow you to automatically create multiple job runs based on a combination of the variables. I took advantage of build matrices to essentially "loop through" the array of analyses and kickoff an instance of the `run_analysis` job for each one.

Nothing uses the `name` variable. I only included it because it will appear in the GitHub Actions workflow logs. `entrypoint` is passed to the `Run Analysis` step below, as are the environment variables.